coffee_maker_navigator_2/lib/features/login/ui/view_model/login_screen_view_model.dart

```diff
@@ -1,11 +1,28 @@
 import 'package:coffee_maker_navigator_2/app/auth/domain/auth_service.dart';
+import 'package:wolt_state_management/wolt_state_management.dart';
 
 class LoginScreenViewModel {
-  final AuthService authService;
+  LoginScreenViewModel({required AuthService authService}) : _authService = authService {
+    _loginState.setIdle(value: false);
+  }
+
+  final AuthService _authService;
+
+  final _loginState = StatefulValueNotifier<bool>.idle(false);
```

Comment:
Let's simplify this by removing the `state` concept. We can simply name this as `isLoggedIn`, and the initial state very well could be `StatefulValueNotifier<bool>.loading()`